controllers/lagoonbuild_controller.go

```diff
@@ -23,6 +23,7 @@ import (
 	"regexp"
 	"sort"
 	"strings"
+    "strconv"
```

Comment:
Formatting nitpick, `go fmt` is a handy thing. If you have vscode there is an extension you can install that will do formatting for you.

Alternatively, `go fmt ./...` or in the case of this repo `make fmt` will do the entire codebase.

This applies to the rest of the code too.

Reply:
TIL. I just pushed a formatting update